src/LibraryManager.Vsix/Microsoft.Web.LibraryManager.Vsix.csproj

```diff
@@ -59,6 +59,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="Commands\InstallLibraryCommand.cs" />
+    <Compile Include="Json\Completion\LibraryIdCompletionEntry.cs" />
```

Comment:
><Compile Include="Json\Completion\LibraryIdCompletionEntry.cs" /> [](start = 4, length = 65)

Place next to others in same directory

Reply:
Yes. will do

---
In reply to: [198546291](https://github.com/aspnet/LibraryManager/pull/258#discussion_r198546291) [](ancestors = 198546291)